From: Chuck Murcko <chuck@telebase.com>
Subject: UnixWare 2.x patch
Affects: Configuration, rfc931.c
ChangeLog: patch to get Apache compiled on UnixWare 2.x, recommended as 
	   a temporary measure, pending rewrite of rfc931.c

*** Configuration.orig	Tue Dec 12 17:43:32 1995
--- Configuration	Tue Dec 12 17:45:26 1995
***************
*** 84,89 ****
--- 84,92 ----
  # For SVR4
  AUX_CFLAGS= -DSVR4
  AUX_LIBS= -lsocket -lnsl -lc
+ # For UnixWare 2.x, no longer just SVR4 (sigh) - use cc, not gcc
+ # AUX_CFLAGS= -DSVR4 -DUW2
+ # AUX_LIBS= -lsocket -lnsl -lcrypt
  # For Amdahl UTS 2.1
  # -Xa enables ANSI mode, -eft is expanded types
  #AUX_CFLAGS= -Xa -eft -DUTS21
*** rfc931.c.orig	Tue Dec 12 17:43:34 1995
--- rfc931.c	Tue Dec 12 17:48:38 1995
***************
*** 85,91 ****
  /* System libraries. */
  
  #include <stdio.h>
! #ifndef QNX
  #include <syslog.h>
  #endif
  #include <sys/types.h>
--- 85,91 ----
  /* System libraries. */
  
  #include <stdio.h>
! #if !defined(QNX) && !defined(UW2)
  #include <syslog.h>
  #endif
  #include <sys/types.h>
***************
*** 123,129 ****
  
  static jmp_buf timebuf;
  
! #ifdef QNX
  
  /*
  Gasp! QNX doesn't support syslog() (out of the box). Replace with output to
--- 123,129 ----
  
  static jmp_buf timebuf;
  
! #if defined(QNX) || defined(UW2)
  
  /*
  Gasp! QNX doesn't support syslog() (out of the box). Replace with output to
***************
*** 132,137 ****
--- 132,142 ----
  
  9 Oct 95
  Ben Laurie <ben@algroup.co.uk>
+ 
+ Same goes for UnixWare 2.x. Sigh.
+ 
+ 12 Dec 95
+ Chuck Murcko <chuck@telebase.com>
  */
  #include <assert.h>
  
***************
*** 149,155 ****
    fprintf(stderr,"%ssystem error %d",logbuf,errno);
  }
  
! #endif /* def QNX */
  
  /* fsocket - open stdio stream on top of socket */
  
--- 154,160 ----
    fprintf(stderr,"%ssystem error %d",logbuf,errno);
  }
  
! #endif /* defined(QNX) || defined(UW2) */
  
  /* fsocket - open stdio stream on top of socket */
  

